thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -65,10 +56,29 @@ export default Component.extend({
    */
   onSelection: null, // function (Set, state)
 
+  context: null,
+
+  compareMode: null, // ""
```

Comment:
It is better to add the type to avoid the type issue.

Reply:
So I get these warnings if I change them to objects.  They are supposed to be passed in by the parent.  What do you recommend?  
<img width="1138" alt="Screen Shot 2021-01-12 at 8 07 48 PM" src="https://user-images.githubusercontent.com/10106733/104405529-07966080-5512-11eb-93b1-25a8860d4cd7.png">

Reply:
We can keep null or undefined because of the warning. I suggest adding the comment for those variables to state the variable expected type.